Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1

```diff
@@ -121,7 +121,7 @@ Invoke-Expression -Command $finalCommand
 #Update Deployment status - https://github.com/projectkudu/kudu/wiki/REST-API#deployment
 
 if($azureWebSite) {
-    $matchedWebSiteName = $azureWebSite.EnabledHostNames | Where-Object { $_ -like '*.scm.azurewebsites.net*' } | Select-Object -First 1
+    $matchedWebSiteName = $azureWebSite.EnabledHostNames | Where-Object { $_ -like '*.scm*azurewebsites.net*' } | Select-Object -First 1
```

Comment:
for app service environments, scm site would be scm.blahblah.p.azurewebsites.net